.travis.yml

```diff
@@ -117,9 +117,9 @@ install:
     if [ "$TRAVIS_OS_NAME" = "osx" ]; then
       # dependencies
       brew update > /dev/null
-      brew install colormake ccache zlib
+      brew install colormake ccache zlib qt
       PATH="/usr/local/opt/ccache/libexec:$PATH"
-      brew link --force zlib
+      brew link --force zlib qt
       brew outdated "pkg-config" || brew upgrade "pkg-config"
```

Comment:
wild guess, could it because `pkg-config` is upgraded after qt is installed?
if so, then better move this line before `brew install`